### DISCUSSION TITLE
Right way to setup miningcore for solo mining.

### DISCUSSION THREAD

**Discussion by babelfish2001** [coinfoundry/miningcore#1293] [Q&A]
I'm looking for suggestions for my config.json file.  I have an ERGO node setup and just want to mine to my node.  I just want the rewards to go to the node  wallet. What should I put in the config.json file for that...  Also if I am mining to my own node, should I use varidiff?  Or should I just set a static difficulty?  Would it be more likely to hit a block with a low static diff like 1.0 if I'm not worrying about network congestion? 

Any help would be appreciated :)

**Comment by snoby** [comment]
@babelfish2001  - did you get an answer on any of your questions on this?

**Comment by seperatis** [comment]
@babelfish2001 @snoby 

One very easy way to do it would be to set the Pool Fee to 100%. 

Perhaps a better way to do it would be to just generate a new wallet address from your Ergo node, `getnewaddress` and mine using that wallet address with the SOLO payment scheme, pool fee set to 0%.

In regards to VarDiff - Variable Difficulty is really just a way to manage network load. It won't have any effect on solving blocks. Static Diff or VarDiff either one. Personally, I would probably set it the way that I always set it....which is based on the ability of my hardware. Look at other mining pools Difficulty and see what they are using for examples.
